app/src/main/java/com/xplore/paymobile/ui/home/HomeFragment.kt

```diff
@@ -109,8 +109,11 @@ class HomeFragment : Fragment() {
                 viewModel.cycleReaders()
             }
             chargeButton.setOnClickListener {
-                val modalBottomSheet = TransactionBottomSheetFragment()
-                modalBottomSheet.show(parentFragmentManager, TransactionBottomSheetFragment.TAG)
+                val modalBottomSheet = TransactionBottomSheetFragment(chargeAmount.toDouble()/100)
```

Comment:
the amount should have two digits after ".", the current method will only show one digit if the amount end with at least one zero